docs/colors/tokens.scss

```diff
@@ -0,0 +1,6 @@
+@use '~@onfido/castor';
+
+:export {
+  @include castor.day();
+  @include castor.night();
```

Comment:
🤓  [no action required]

I guess merging is fine, but so would be only including one, assuming day and night always have the same structure and only have values changed.